app/models/fact_check.rb

```diff
@@ -41,6 +42,15 @@ def language_in_allowed_values
     errors.add(:language, I18n.t(:"errors.messages.invalid_article_language_value")) unless allowed_languages.include?(self.language)
   end
 
+  def rating_in_allowed_values
+    core_statuses = YAML.load(ERB.new(File.read("#{Rails.root}/config/core_statuses.yml")).result)
+    core_keys = core_statuses["MEDIA_CORE_VERIFICATION_STATUSES"].collect{|s| s[:id]}
+    custom_statuses = self.claim_description.team&.get_media_verification_statuses
+    custom_keys = custom_statuses['statuses'].collect{|s| s[:id]} unless custom_statuses.blank?
+    allowed_statuses = custom_keys || core_keys
```

Comment:
This looks like too much copy-and-paste from  other places, and too much status-specific logic to belong here. Can we call `team.verification_statuses('media', nil)`? If so, please confirm it works for core statuses and custom statuses.

Reply:
Done